mantle/kola/harness.go

```diff
@@ -551,13 +552,31 @@ func runProvidedTests(tests map[string]*register.Test, patterns []string, multip
 			reporters.NewJSONReporter("report.json", pltfrm, versionStr),
 		},
 	}
+	var confs []*conf.UserData
+	for _, test := range tests {
+		if test.NonExclusive {
+			confs = append(confs, test.UserData)
+		}
+	}
+
+	mergedConfig, err := conf.MergeUserData(confs)
+	if err != nil {
+		plog.Fatalf("Error merging configs: %v", err)
+	}
+	err = runNonExclusiveTests(opts, tests, flight, *mergedConfig, outputDir)
```

Comment:
Hmm, I think instead we'd want to add it to the harness as just another test so that it runs in parallel with the other exclusive tests, no?

The tricky bit then is reporting I think.

Hmm OK, so I think the question here is whether we drive down this new concept into the `Suite` API or not. If we do, then we can get better reporting because e.g. we'd have individual `PASS` lines instead of a single one, though it'd make the code more complex.

Ahh, I think we can leverage subtests for this. So then here, all we'd do is create a *new* test, which has for its `Run` just a for-loop where we run all the tests using `c.Run(testName, test.Run)`. That should allow you to drop `runNonExclusiveTest[s]`.

In a future step, we can integrate this better into reporting.